codice-itest-impl/src/main/java/org/codice/itest/reporter/TestReporterConfiguration.java

```diff
@@ -52,22 +35,22 @@ public ExitCodeReporter exitCodeReporter(){
     }
 
     @Bean("testReporter")
-    @ConditionalOnProperty(prefix="itest.reporter", name="logger")
+    @ConditionalOnProperty(prefix="codice.itest", name="loggerName")
     public Consumer<TestResult> loggingDiagnosticTestReporter() {
-        Logger logger = LoggerFactory.getLogger(loggerName);
-        return new LoggingDiagnosticTestReporter(logger, exitCodeReporter(), toStringFormatter());
+        System.out.println(iTestConfigurationProperties.toString());
```

Comment:
❓ Is this System.out intentional or leftover from debugging?

Reply:
yeah, leftover.  I'll remove it.